AutoDI/TypeKeyNotFoundEventArgs.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace AutoDI
+{
+    public class TypeKeyNotFoundEventArgs : EventArgs
+    {
+        Type Key_;
+        DateTime EventTime_;
```

Comment:
Fields should be prefixed with an underscore and pascal case, and in these cases should be marked `readonly`
```C#
private readonly Type _key;
```

Reply:
Done